pkg/addons/data/ClusterRoleBinding.rbac.authorization.k8s.io/customer-admin-project-group.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
don't think this file should be here? customer-admins should get a rolebinding to customer-admin-project for each project?

Reply:
Will test.

Reply:
Will remove.